xwiki-platform-core/xwiki-platform-extension/xwiki-platform-extension-index/src/main/java/org/xwiki/extension/index/security/SecurityVulnerabilityDescriptor.java

```diff
@@ -39,6 +42,8 @@ public class SecurityVulnerabilityDescriptor
 {
     private String id;
```

Comment:
Should we also change all the reference to the very generic "id" naming to something closer to "cve" or "cveId" ?